lib/Doctrine/ORM/Query/Printer.php

```diff
@@ -13,28 +13,16 @@
  */
 class Printer
 {
-    /**
-     * Current indentation level
-     *
-     * @var int
-     */
-    protected $_indent = 0;
-
-    /**
-     * Defines whether parse tree is printed (default, false) or not (true).
-     *
-     * @var bool
-     */
-    protected $_silent;
+    /** Current indentation level */
+    protected int $_indent = 0;
 
     /**
      * Constructs a new parse tree printer.
      *
-     * @param bool $silent Parse tree will not be printed if true.
+     * @param bool $_silent Parse tree will not be printed if true.
      */
-    public function __construct($silent = false)
+    public function __construct(protected $_silent = false)
```

Comment:
```suggestion
    public function __construct(protected bool $_silent = false)
```

The PR is already labeled as a BC break. Isn't this also a good time to remove the underscore prefix?

Reply:
I will do a separate, global PR if you don't mind.

Reply:
See https://github.com/doctrine/orm/pull/10500

Reply:
I think we've discussed similar changes already: I wouldn't switch parameter names to the discouraged underscore notation just for the sake of using CPP.

Reply:
Then approve #10500. Or approve this one and let me rebase #10500, the end result will be the same: CPP and no underscore.

Reply:
Actually I just pushed a version with no underscores so that we can deal with both PRs separately. More simple for everyone.